src/lib/virtio/net_device.lua

```diff
@@ -205,6 +220,7 @@ function VirtioNetDevice:tx_packet_start(addr, len)
 
    -- TODO: copy the relevnat fields from the packet
```

Comment:
This TODO can disappear now? (Maybe worth sweeping for more comments that we have obsoleted?)